framework/src/Volo.Abp.Ddd.Domain/Volo/Abp/Domain/Repositories/RepositoryBase.cs

```diff
@@ -85,6 +85,12 @@ public IEnumerator<TEntity> GetEnumerator()
 
         public abstract Task DeleteAsync(Expression<Func<TEntity, bool>> predicate, bool autoSave = false, CancellationToken cancellationToken = default);
 
+        public virtual Task DeleteManyAsync(
+            Expression<Func<TEntity, bool>> predicate,
+            bool autoSave = false,
+            CancellationToken cancellationToken = default)
+            => DeleteAsync(predicate, autoSave, cancellationToken);
+
```

Comment:
I'm not sure if this method should be included because I have made it so that `DeleteAsync(Expression..` will automatically call DeleteManyAsync if it can.

Reply:
Actually, maybe the DeleteAsync method should be made obsolete instead because in reality it is actually capable of deleting many entities.

Reply:
Removed https://github.com/abpframework/abp/pull/7543/commits/f4458ac7d039c531dfe0b5dbdd91ddbd121eded5